_usage/limitations-of-babelfish.md

```diff
@@ -158,16 +159,17 @@ features are discovered, and additional features are added to MS SQL:
 | JSON | Datatypes, Built-in Functions, and statements are unsupported. |
 | `KILL` | This syntax is not supported. |
 | `CREATE/ALTER/DROP EXTERNAL LANGUAGE` | This syntax is not supported. |
-| `CREATE/ALTER/DROP EXTERNAL LIBRARY` | Functionality related to this object type is not supported |
+| `CREATE/ALTER/DROP EXTERNAL LIBRARY` | Functionality related to this object type is not supported. |
+| Localization of messages | Only English error messages and date names are currently supported. |
 | `CREATE/ALTER LOGIN` clauses are supported with limited syntax | The `CREATE LOGIN ... PASSWORD` clause, `...DEFAULT_DATABASE` clause, and `...DEFAULT_LANGUAGE` clause are supported. The `ALTER LOGIN ... PASSWORD` clause is supported, but `ALTER LOGIN ... OLD_PASSWORD` clause is not supported. Only a sysadmin login user can modify a password. |
 | `LOGIN` objects | All options for `LOGIN` objects except: `PASSWORD`, `DEFAULT_DATABASE`, `ENABLE`, `DISABLE` |
 | `LOGINPROPERTY()` function | This function is not supported. |
-| `CREATE/ALTER/DROP OPEN/CLOSE, BACKUP/RESTORE MASTER KEY`, | Functionality related to this syntax is not supported |
+| `CREATE/ALTER/DROP OPEN/CLOSE, BACKUP/RESTORE MASTER KEY`, | Functionality related to this syntax is not supported. |
 | Materialized Views | Materialized views are not supported. |
 | `CREATE/ALTER/DROP MESSAGE TYPE`| This syntax is not supported. |
 | `NEWSEQUENTIALID()` function | Implemented as NEWID(); sequential behavior is not supported. |
 | `MERGE` | This syntax is not supported. |
-| `NEWSEQUENTIALID()` function | When calling `NEWSEQUENTIALID()`, PostgreSQL can\'t guarantee a higher GUID value so it will just generate a new GUID value, just like `NEWID()` does. |
+| `NEWSEQUENTIALID()` function | When calling `NEWSEQUENTIALID()`, PostgreSQL can't guarantee a higher GUID value so it will simply generate a new GUID value, just like `NEWID()` does. |
```

Comment:
avoid words like 'just' or 'simply' they do not add any value here.

Reply:
I added a missing comma and used "instead" rather than "simply".
I'll leave the "just", however, because that sounds better to me.
The sentence now reads:
"When calling `NEWSEQUENTIALID()`, PostgreSQL cannot guarantee a higher GUID value, so it will instead generate a new GUID value, just like `NEWID()` does."